ci/authn-gce/get_gce_id_tokens.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
+
+# This script obtains all of the GCE identity tokens required by authn-gce cucumber tests
+# The scripts accepts the GCE instance name as an argument and an optional GCP zone (default is us-central1-a).
+# The script does the following:
+# 1. Validates gcloud is installed (requires a GCP account)
+#   To install gcloud goto: https://cloud.google.com/sdk/docs, download and install the SDK by running:
+#   ./google-cloud-sdk/install.sh
+#   ./google-cloud-sdk/bin/gcloud init
+# 2. Verifies the GCE instance exists and is in status running
+# 3. Executes a ssh curl command and write the output token with the appropriate token name to '../ci/authn-gce/tokens'.
+
+PROGNAME=$(basename "$0")
+INSTANCE_ZONE=us-central1-a
```

Comment:
you said that you succeeded to work without the zone. is this still needed? if so, why?

Reply:
Yes you cannot execute `gcloud compute ssh` without a zone, I'm extracting the instance zone.

Reply:
then why do we have this value here?

Reply:
The value has no meaning, the variable is used throughout the script.

Reply:
so can we remove the value and define it as an empty string? reading this like it is makes me think that the value is used.